core/cibox-project-builder/files/vagrant/box/provisioning/ansible/run-ansible-playbook.sh

```diff
@@ -2,17 +2,19 @@
 
 export PYTHONUNBUFFERED=1
 
+# Commented playbook stay here just in case you'll need it for updating an image or
+# specific software settings.
 playbooks=(
-/vagrant/provisioning/ansible/playbooks/sendmail.yml \
+# /vagrant/provisioning/ansible/playbooks/sendmail.yml \
 /vagrant/provisioning/ansible/playbooks/misc.yml \
 /vagrant/provisioning/ansible/playbooks/www.yml \
-/vagrant/provisioning/ansible/playbooks/php.yml \
-/vagrant/provisioning/ansible/playbooks/php-xdebug.yml \
-/vagrant/provisioning/ansible/playbooks/mysql.yml \
-/vagrant/provisioning/ansible/playbooks/memcached.yml \
+# /vagrant/provisioning/ansible/playbooks/php.yml \
+# /vagrant/provisioning/ansible/playbooks/php-xdebug.yml \
+# /vagrant/provisioning/ansible/playbooks/mysql.yml \
+# /vagrant/provisioning/ansible/playbooks/memcached.yml \
```

Comment:
It is like "optional", we change php version so rare. About mysql I can't recall last time we were making changes, php-xdebug never touched, memcached as well